scripts/pre_commit_linter.py

```diff
@@ -470,22 +470,19 @@ def _redirect_stdout(new_target):
         sys.stdout = old_target
 
 
-def _get_all_files():
-    """This function is used to parse the arguments entered by the user, and
-    returns the list of all files for linting and pattern checks.
+def _get_all_files(input_files, input_path):
+    """This function is used to return the files which needs to linted and
+    checked.
 
+    Args:
+        input_files: list. The list of files to be linted and checked.
```

Comment:
Explain that this is ignored if input_path is specified. Also, put this as the second arg (in the function signature too) since it seems to have second priority.

Reply:
Done